security/azure/pg_conn_throttling/pg_conn_throttling.pt

```diff
@@ -0,0 +1,193 @@
+      field "subscriptionName", jmes_path(col_item,"displayName")
+    end
+  end
+end
+
+datasource "ds_pg_servers" do
+  iterate $ds_subscriptions
+  request do
+    auth $azure_auth
+    pagination $azure_pagination
+    host $param_azure_endpoint
+    path join(["/subscriptions/", val(iter_item, "subscriptionId"), "/providers/Microsoft.DBForPostgreSql/servers"])
+    query "api-version", "2017-12-01"
+    header "User-Agent", "RS Policies"
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response, "value[*]") do
+      field "id", jmes_path(col_item, "id")
+      field "name", jmes_path(col_item, "name")
+      field "sku", jmes_path(col_item, "sku")
+      field "location", jmes_path(col_item, "location")
+      field "properties", jmes_path(col_item, "properties")
+      field "tags", jmes_path(col_item, "tags")
+      field "subscriptionId", val(iter_item, "subscriptionId")
+      field "subscriptionName", val(iter_item, "subscriptionName")
+    end
+  end
+end
+
+datasource "ds_pg_servers_throttling" do
+  iterate $ds_pg_servers
+  request do
+    auth $azure_auth
+    pagination $azure_pagination
+    host $param_azure_endpoint
+    path join([val(iter_item, "id"), "/configurations/connection_throttling"])
+    query "api-version", "2017-12-01"
+    header "User-Agent", "RS Policies"
+  end
+  result do
+    encoding "json"
+    field "id", val(iter_item, "id")
+    field "name", val(iter_item, "name")
+    field "sku", val(iter_item, "sku")
+    field "location", val(iter_item, "location")
+    field "properties", val(iter_item, "properties")
+    field "tags", val(iter_item, "tags")
+    field "subscriptionId", val(iter_item, "subscriptionId")
+    field "subscriptionName", val(iter_item, "subscriptionName")
+    field "configName", jmes_path(response, "name")
+    field "configProperties", jmes_path(response, "properties")
+  end
+end
+
+datasource "ds_bad_pg_servers" do
+  run_script $js_bad_pg_servers, $ds_pg_servers_throttling
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_bad_pg_servers", type: "javascript" do
+  parameters "ds_pg_servers"
+  result "result"
+  code <<-EOS
+  result = []
+
+  _.each(ds_pg_servers, function(server) {
+    if (server['configProperties']['value'] != 'on') {
+      result.push(server)
+    }
+  })
+EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_pg_servers_throttling" do
+  validate $ds_bad_pg_servers do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Azure PostgreSQL Server(s) Found Without Connection Throttling Enabled"
+    escalate $esc_email
+    check eq(size(data),0)
+    export do
+      field "name" do
+        label "Name"
```

Comment:
Name should be Postgres Instance name or something similar.

Reply:
It's implied that all of the fields are for a Postgres instance by the subject/summary:

Azure PostgreSQL Server(s) Found Without Connection Throttling Enabled